### ISSUE TITLE
java.lang.NoClassDefFoundError in version 3.0, Jenkins 2.121.3

### ISSUE THREAD

**Issue by el-psycho** [jenkinsci/github-autostatus-plugin#12]
Getting the following error on version 3.0 with Jenkins 2.121.3. Note that `org/jenkinsci/plugins/pipeline` changes some times depending on the job but the rest of the error looks the same. Tried downgrading to 2.1 but that didnt make the error go away. I guess some configuration or something must remain. I had to completely uninstall this plugin to get my jenkins jobs working again:

```
java.lang.NoClassDefFoundError: org/jenkinsci/plugins/pipeline/modeldefinition/shaded/com/google/common/base/Strings
	at org.jenkinsci.plugins.githubautostatus.notifiers.InfluxDbNotifier.<init>(InfluxDbNotifier.java:61)
	at org.jenkinsci.plugins.githubautostatus.notifiers.BuildNotifierManager.addInfluxDbNotifier(BuildNotifierManager.java:73)
	at org.jenkinsci.plugins.githubautostatus.BuildStatusAction.addInfluxDbNotifier(BuildStatusAction.java:107)
	at org.jenkinsci.plugins.githubautostatus.GithubBuildStatusGraphListener.checkEnableBuildStatus(GithubBuildStatusGraphListener.java:251)
	at org.jenkinsci.plugins.githubautostatus.GithubBuildStatusGraphListener.onNewHead(GithubBuildStatusGraphListener.java:72)
	at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.notifyListeners(CpsFlowExecution.java:1438)
	at org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$3.run(CpsThreadGroup.java:417)
	at org.jenkinsci.plugins.workflow.cps.CpsVmExecutorService$1.run(CpsVmExecutorService.java:35)
	at hudson.remoting.SingleLaneExecutorService$1.run(SingleLaneExecutorService.java:131)
	at jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)
	at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by jeffpearce** [comment]
Looks like a dependency (which we shouldn't have had in the first place) went away. This seems like an easy fix

**Comment by jeffpearce** [comment]
Thanks for reporting this @el-psycho. 

This should be fixed in release 3.0.1, which will be available sometime today. Please let me know if you continue to have issues.
